WalletContext/src/chains.ts

```diff
@@ -0,0 +1,204 @@
+import type { AddEthereumChainParameter } from '@web3-react/types'
+
+const INFURA_KEY = process.env.REACT_APP_INFURA_KEY
+  ? process.env.REACT_APP_INFURA_KEY
+  : process.env.NEXT_PUBLIC_APP_INFURA_KEY
+
+const ALCHEMY_KEY = process.env.REACT_APP_ALCHEMY_KEY
```

Comment:
would these be better as props? it seems like a bit of a bad pattern to expect that whatever imports this library now needs to set these exact ENV vars. 

If not, this expectation should be make very clear. Docs., env.example ect

Reply:
![image](https://user-images.githubusercontent.com/3683356/180121854-90d029c1-a990-4505-b9ae-54117479291b.png)

ive tried passing the env variables as props but it ended up throwing 401 errors. 

will just make it clear in docs and .env example that infura and alchemy key needs to be passed